tests/cases/debugger_hlsl_line_stepping.amber

```diff
@@ -25,35 +40,7 @@ SHADER vertex vtex_shader SPIRV-ASM
      %DbgExt = OpExtInstImport "OpenCL.DebugInfo.100"
                OpMemoryModel Logical GLSL450
                OpEntryPoint Vertex %main "main" %pos %color %gl_Position %out_var_COLOR
-        %src = OpString "simple_vs.hlsl"
-       %code = OpString "
-struct VS_OUTPUT {
-  float4 pos : SV_POSITION;
-  float4 color : COLOR;
-};
-
-VS_OUTPUT main(float4 pos : POSITION,
-               float4 color : COLOR) {
-  VS_OUTPUT vout;
-  vout.pos = pos;
-  vout.color = color;
-  return vout;
-}
-"
-               OpSource HLSL 600 %src "
```

Comment:
This lost the OpSource, was that intentional?

Reply:
Yes. The content is now in the `VIRTUAL_FILE`.
We want to test that we're grabbing the file content from the virtual file, not the embedded source.